### PR TITLE
Add DB::delete method

### PR DESCRIPTION
**Pull request by jf-viguier**
Add delete method and some text to explain

| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Please be specific when describing the PR. What did you add, why did you submit it?
| Fixed ticket? | Fixes #{issue number here} if there is a related issue

### CODE REVIEW

File: src/content/1.7/development/database/db.md [hunk 1]
```diff
@@ -135,3 +135,16 @@ $result = $db->update('db_table', array(
...
+```php
+<?php
+/** @var bool $result */
+$result = $db->delete('db_table', 'id_table = 10');
+```
+Method signature: delete($table, $where = '', $limit = 0, $use_cache = true, $add_prefix = true)
+
+`_DB_PREFIX_` will be automatically prefixed to the table name if $add_prefix is true (by default).
+
+The result is boolean saying if the request was properly executed or not.
```
Comment (Progi1984) [inline src/content/1.7/development/database/db.md:150]:
```suggestion
The result is a boolean saying if the request was properly executed or not.
```

File: src/content/1.7/development/database/db.md [hunk 2]
```diff
@@ -135,3 +135,16 @@ $result = $db->update('db_table', array(
...
 The result is boolean saying if the request was properly executed or not.
+
+### Delete rows in the database
+
+```php
+<?php
+/** @var bool $result */
+$result = $db->delete('db_table', 'id_table = 10');
+```
+Method signature: delete($table, $where = '', $limit = 0, $use_cache = true, $add_prefix = true)
```
Comment (PierreRambaud) [inline src/content/1.7/development/database/db.md:146]:
```suggestion
Method signature: `delete($table, $where = '', $limit = 0, $use_cache = true, $add_prefix = true)`
```

File: src/content/1.7/development/database/db.md [hunk 3]
```diff
@@ -135,3 +135,16 @@ $result = $db->update('db_table', array(
...
+### Delete rows in the database
+
+```php
+<?php
+/** @var bool $result */
+$result = $db->delete('db_table', 'id_table = 10');
+```
+Method signature: `delete($table, $where = '', $limit = 0, $use_cache = true, $add_prefix = true)`
+
+`_DB_PREFIX_` will be automatically prefixed to the table name if $add_prefix is true (by default).
```
Comment (Progi1984) [inline src/content/1.7/development/database/db.md:148]:
```suggestion
`_DB_PREFIX_` will be automatically prefixed to the table name if `$add_prefix` is `true` (by default).
```

**Review by Progi1984 — COMMENTED** [review]
Small feedbacks

**Review by Progi1984 — APPROVED** [review]

**Review by matthieu-rolland — COMMENTED** [review]
LGTM for the improvement, but there is a conflict to solve on this file

**Comment by jf-viguier** [comment]
@matthieu-rolland conflict resolved

**Comment by Progi1984** [comment]
@jf-viguier Could you rebase your PR ? Merge commits are not allowed.

**Comment by jf-viguier** [comment]
@Progi1984 rebased

**Review by Progi1984 — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thank you @jf-viguier

**Pull request merged**